### ISSUE TITLE
namedtuple no-member false positives

### ISSUE THREAD

**Issue by pylint-bot** [PyCQA/pylint#92]
Originally reported by: **Jonathan Kotta (BitBucket: [jpkotta](http://bitbucket.org/jpkotta), GitHub: @jpkotta?)**

---

If a namedtuple is defined with a string,  there must be whitespace around each member.  In particular, a namedtuple definition with a comma-separated string doesn't work.

Also, the standard methods _make(), _asdict(), and _fields() are not recognized.

```
$ pylint --version
No config file found, using default configuration
pylint 1.0.0, 
astroid 1.0.0, common 0.60.0
Python 3.3.2 (default, Sep  6 2013, 09:30:10) 
[GCC 4.8.1 20130725 (prerelease)]
```

astroid is actually not the released 1.0.0, but this rev: https://bitbucket.org/logilab/astroid/commits/5ed6266cab789bc89d882444c938f03953491496

---
- Bitbucket: https://bitbucket.org/logilab/pylint/issue/92
